src/components/Form.jsx

```diff
@@ -1,4 +1,4 @@
-import { isEqual } from 'lodash'
+import { fromJS, is } from 'immutable'
```

Comment:
I generally don't think `immutable` should be visible in components. I think we should write a helper function called `isEqual` that abstracts this away for us.